code/__DEFINES/mode.dm

```diff
@@ -130,10 +135,11 @@ var/global/list/ROLES_SPECIAL = list(JOB_SURVIVOR)
 
 var/global/list/ROLES_USCM = ROLES_CIC + ROLES_POLICE + ROLES_AUXIL_SUPPORT + ROLES_MISC + ROLES_ENGINEERING + ROLES_REQUISITION + ROLES_MEDICAL + ROLES_MARINES - ROLES_WO
 var/global/list/ROLES_GROUND = ROLES_XENO + ROLES_SPECIAL + ROLES_WHITELISTED
+var/global/list/ROLES_MUDSKIPPERS = MUDSKIPPER_COMMAND + MUDSKIPPER_SQUAD + MUDSKIPPER_SUPPORT + MUDSKIPPER_WYREPS + ROLES_ENGINEERING + ROLES_REQUISITION - ROLES_WO
 
 var/global/list/ROLES_DISTRESS_SIGNAL = ROLES_USCM + ROLES_GROUND
 var/global/list/ROLES_FACTION_CLASH = ROLES_USCM + JOB_PREDATOR
-
+var/global/list/ROLES_NEROID = ROLES_MUDSKIPPERS + JOB_PREDATOR
```

Comment:
var/global/list/ROLES_NEROID = list(ROLES_MUDSKIPPERS,JOB_PREDATOR)

is the only significant difference I see.

Reply:
You can aslo try a single element list 
var/global/list/ROLES_NEROID = list(ROLES_MUDSKIPPERS) and see if its the bredators that are fucking you up. Stranger things have happened.

Reply:
If the above fails, do what I originally suggested on the discord. Change all var/global/list declarations to proper list() statements. I still think that's what messing you up, somewhere it loses the list format for some reason, even though I see no reason why it would.